### PR TITLE
JW-73 Implement missing functions for ini_parser.c

### PR DESCRIPTION
**Pull request by mmasztalerczuk**
Remove TODOs:
   * utils/ini_parser.c:162:// TODO: Implement read_ip for windows platform
   * utils/ini_parser.c:178: // TODO: Implement get_domain for windows platform
   * utils/ini_parser.c:192: // TODO: Implement get_typ for windows platform
   * utils/ini_parser.c:224: // TODO: Implement get_protocol for windows platform

### CODE REVIEW

File: utils/ini_parser.c [hunk 1]
```diff
@@ -159,45 +160,40 @@ read_string(const char *section, const char *key)
...
-	/**
     const char *platform = read_string(DEFAULT_SECTION, PLATFORM_KEY);
     if (platform &&
        (strcmp(platform, PLATFORM_DPDK) == 0 ||
         strcmp(platform, PLATFORM_NIC) == 0)) {
         return AF_INET;
-    } */
-	return -1337;//AF_NETLINK;
+    }
+	return AF_NETLINK;
```
Comment (michal-clapinski) [inline utils/ini_parser.c]: tab instead of spaces?

File: windows/include/vr_utils_temp.h [hunk 2]
```diff
@@ -2,11 +2,30 @@
...
+typedef INT16 int16_t;
+typedef UINT16 uint16_t;
+typedef INT32 int32_t;
+typedef UINT32 uint32_t;
+typedef INT64 int64_t;
+typedef UINT64 uint64_t;
+
 #define IFNAMSIZ 16
-#define AF_BRIDGE 7
+#define AF_BRIDGE    7
```
Comment (michal-clapinski) [inline windows/include/vr_utils_temp.h]: tab instead of space?

File: windows/include/vr_utils_temp.h [hunk 3]
```diff
@@ -2,11 +2,30 @@
...
 
+#define __attribute__packed__open__ __pragma( pack( push, 1 ) )
+#define __attribute__packed__close__ __pragma( pack( pop ) )
+#define __attribute__format__open__(...) /* do nothing */
+#define __attribute__format__close__(...) /* do nothing */
+
+// TODO: Remove unused defines and structures
 #define __attribute__(A) /* do nothing */
 
 #define NLM_F_REQUEST           1       /* It is request message.       */
```
Comment (michal-clapinski) [inline windows/include/vr_utils_temp.h]: I don't understand this comment. Could you clarify it?

Comment (mmasztalerczuk) [inline windows/include/vr_utils_temp.h]: This is not my comment. It will be removed in future (there is TODO for that)

**Pull request merged**